app/views/emails/_email_form.erb

```diff
@@ -3,11 +3,11 @@
       <label class="control-label col-sm-2" for="hide_show_recipients"><%= t('.further_recipients') %>:</label>
       <div class="col-sm-6 btn-group" data-toggle="buttons">
         <label class="btn btn-default <%= 'active' unless @email.hide_recipients %>">
-          <%= f.radio_button :hide_recipients, @email.hide_recipients, checked: !@email.hide_recipients %>
+          <%= f.radio_button :hide_recipients, @email.hide_recipients = false %>
           <%= t('.show_recipients') %>
         </label>
         <label class="btn btn-default <%= 'active' if @email.hide_recipients %>">
-          <%= f.radio_button :hide_recipients, !@email.hide_recipients, checked: @email.hide_recipients%>
+          <%= f.radio_button :hide_recipients, @email.hide_recipients = true%>
```

Comment:
Same as above.
Further, isnt @email.hide_recipients always true after the view was rendered? I dont really understand what you are doing here